lib/jsdom/living/nodes/Node.webidl

```diff
@@ -15,9 +15,11 @@ interface Node : EventTarget {
   readonly attribute unsigned short nodeType;
   readonly attribute DOMString nodeName;
 
-  readonly attribute DOMString? baseURI;
+  readonly attribute USVString baseURI;
```

Comment:
Not sure if a `DOMString` -> `USVString` change is worth mentioning in the changelog, but highlighting this just in case.